docker/run-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+NAME=$(basename $(dirname `pwd`))
+
+CONTAINER=`docker ps -a -q --filter="name=$NAME"`
+if [ -n "$CONTAINER" ]; then
+    docker rm $CONTAINER
+fi
+
+CONTAINER=`docker ps -a -q --filter="name=$NAME"`
+
+docker run -it \
+    --name $NAME \
+    --init \
+    -v $HOME/.clasprc.json:/root/.clasprc.json \
+    -v `pwd`/../src:/usr/src/app/src \
```

Comment:
actually update the container to use /src instead of this longer /usr/src/app/src path